src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py

```diff
@@ -781,6 +782,26 @@ def hcells_list(self, value: List[str]) -> None:
         self.attr_setter("_hcells_list", value)
 
 
+    @property
+    def test_attr(self) -> str:
+        """
+        Get the foobar.
```

Comment:
Looks like some test code snuck into the actual implementation

Reply:
Oops.